pytorch_lightning/strategies/fully_sharded.py

```diff
@@ -107,6 +108,7 @@ def __init__(
             cluster_environment=cluster_environment,
             checkpoint_io=checkpoint_io,
             precision_plugin=precision_plugin,
+            pg_backend=pg_backend,  # TODO: Remove inheritance from DDPStrategy
```

Comment:
How does the added TODOs relate to these changes?

Reply:
they're not, but are needed for overall API stability. the only reason these arguments extend from DDP is for subprocess launching. I can remove the TODOs and make that a separate issue.

Reply:
Yeah, I would prefer that as there is more nuance to what this TODO says